### PR TITLE
add locales dsb, hsb, sw to files

### PR DESCRIPTION
**Pull request by peiying2**
Bugs 1367180, 1367179, and 1367162

### CODE REVIEW

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -350,7 +350,7 @@
             'en-ZA', 'es-AR', 'es-CL', 'es-ES', 'es-MX', 'eu', 'fa', 'fr',
             'fy-NL', 'hi-IN', 'hsb', 'it', 'ja', 'kab', 'km', 'ko', 'lt', 'ms',
             'ncj', 'nl', 'nv', 'pl', 'pt-BR', 'pt-PT', 'ro', 'ru', 'son', 'sq',
-            'sr', 'sv-SE', 'tr', 'uk', 'ur', 'zam', 'zh-CN', 'zh-TW',
+            'sr', 'sv-SE', 'sw', 'tr', 'uk', 'ur', 'zam', 'zh-CN', 'zh-TW',
```
Comment (flodolo) [inline app/config/sources.inc.php]: Let's not add it here, it's an extremely old page.

File: app/config/sources.inc.php [hunk 2]
```diff
@@ -404,7 +404,7 @@
             'es-MX', 'eu', 'fa', 'fi', 'fr', 'fy-NL', 'gd', 'gl', 'hi-IN', 'hr',
             'hsb', 'hu', 'id', 'it', 'ja', 'kab', 'km', 'ko', 'lt', 'mk', 'ms',
             'ncj', 'nl', 'nv', 'pl', 'pt-BR', 'pt-PT', 'ro', 'ru', 'sk', 'sl',
-            'son', 'sq', 'sr', 'sv-SE', 'tr', 'uk', 'uz', 'vi', 'xh', 'zam',
+            'son', 'sq', 'sr', 'sv-SE', 'sw', 'tr', 'uk', 'uz', 'vi', 'xh', 'zam',
```
Comment (flodolo) [inline app/config/sources.inc.php]: I'm starting to think that we should open this to all mozilla.org locales.

File: app/config/sources.inc.php (hunk 1 above)
Comment (peiying2) [inline app/config/sources.inc.php]: should i remove some other recently added locales, am, ncj, nv

File: app/config/sources.inc.php (hunk 2 above)
Comment (peiying2) [inline app/config/sources.inc.php]: changed to 
'supported_locales' => $mozilla,

File: app/config/sources.inc.php (hunk 1 above)
Comment (flodolo) [inline app/config/sources.inc.php]: No, at this point I would leave them be.

**Comment by peiying2** [comment]
@flodolo sw has been in the supported locale list a while back. Is that what you are referring to?

**Pull request merged**